resources/public/api/conf/3.0/schemas/retrieve/calculation/businessProfitAndLoss.json

```
@@ -1,13 +1,225 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
-  "description": "###TODO###",
+  "description": "A statement summarising the profit and loss position for business income sources. The enum in the incomeSourceTypes will determine the business type. When the type is Self Employment totalExpenses and totalAdditions may be negative amounts. Other business sources will continue to return positive amounts.",
   "type": "object",
   "properties": {
-    "###TODO###": {
-      "description": "Other properties",
+    "incomeSourceId": {
+      "title": "incomeSourceId",
+      "description": "The unique identifier of the income source.",
+      "type": "string",
+      "pattern": "^[A-Za-z0-9]{15}$",
+      "example": "XAIS12345678910"
+    },
+    "incomeSourceType": {
+      "title": "incomeSourceType",
+      "description": "The income source type.",
+      "type": "string",
+      "enum": [
+        "self-employment",
+        "uk-property-non-fhl",
+        "foreign-property-fhl-eea",
+        "uk-property-fhl",
+        "foreign-property"
+      ]
+    },
+    "incomeSourceName": {
+      "title": "incomeSourceName",
+      "description": "The unique identifier of the income source.",
       "type": "string"
+    },
+    "totalIncome": {
+      "title": "totalIncome",
+      "description": "The total income used in the business calculation. This value must be between 0 and 99999999999.99 up to 2 decimal places.",
+      "type": "number",
+      "multipleOf": 0.01,
+      "minimum": 0,
+      "maximum": 99999999999.99,
+      "example": 160
```

All `"example"` field values need to be quoted as strings:

```suggestion
      "example": "160"
```